osp-deploy/cinder/README.md

```diff
@@ -200,6 +201,50 @@ parameter_defaults:
         value: 'cindervol'
     cinder_user_enabled_backends: ['tripleo_dellemc_dellsc']
 ```    
+**4. PowerStore iSCSI and FC drivers**  
+For full detailed instruction of options please refer to [PowerStore Backend Configuration](https://docs.openstack.org/cinder/latest/configuration/block-storage/drivers/dell-emc-powerstore-driver.html#driver-configuration)
+
+**Environment sample**
+
+With a director deployment, PowerStore backend can be deployed using the integrated heat environment file. This file is located in the following path of the Undercloud node:
+/usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-powerstore-config.yaml
+
+Copy this file to a local path where you can edit and invoke it later. For example, to copy it to ~/templates/:
+
+```bash
+$ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-powerstore-config.yaml ~/templates/
+```
+Afterwards, open the copy (~/templates/cinder-dellemc-powerstore-config.yaml) and edit it as you see fit. The following shows a sample content of the file. The files will list optional params that the user can choose to override if they don't like the default value.
+
+```yaml
+# A Heat environment file which can be used to enable a
+# Cinder Dell EMC PowerMax backend, configured via puppet.
+resource_registry:
+  OS::TripleO::Services::CinderBackendDellEMCPowerStore: ../deployment/cinder/cinder-backend-dellemc-powerstore-puppet.yaml
+
+parameter_defaults:
+  CinderEnablePowerStoreBackend: true
+  CinderPowerStoreBackendName: 'tripleo_dellemc_powerstore'
+  CinderPowerStoreMultiConfig: {}
+  CinderPowerStoreAvailabilityZone: ''
+  CinderPowerStoreSanIp: '10.10.10.10'
+  CinderPowerStoreSanLogin: 'admin'
+  CinderPowerStoreSanPassword: 'password'
+  CinderPowerStoreAppliances: 'appliance-1,appliance2'
+  CinderPowerStorePorts: '10.10.10.11,10.10.10.12'
+  CinderPowerStoreStorageProtocol: 'iSCSI'
+
+# To configure multiple PowerStore backends, use CinderPowerStoreMultiConfig to
```

Comment:
example above already covers two powerstores.

#To configure multiple PowerStore backends, use CinderPowerStoreMultiConfig to
# assign parameter values specific to that backend. For example:
->
#To configure multiple PowerStore backends differently, use CinderPowerStoreMultiConfig to
# assign parameter values specific to each backend. For example:

Reply:
Ivan, can you make these small changes so we can merge the patch?